src/utils/parse.ts

```diff
@@ -0,0 +1,51 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// Tests are already submitted in https://github.com/mozilla/blurts-server/pull/3359:
+/* c8 ignore start */
+
+type E164PhoneNumberString = string | null;
```

Comment:
In the original `.js` file, you can see this [JSDoc comment](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#typedef-callback-and-param):

```js
/**
 * @typedef {string} ISO8601DateString See https://en.wikipedia.org/wiki/ISO_8601
 * @typedef {`+${string}`} E164PhoneNumberString See https://en.wikipedia.org/wiki/E.164
 */
```

The TypeScript equivalent would be:

```ts
/** See https://en.wikipedia.org/wiki/ISO_8601 */
export type ISO8601DateString = string;
/**  See https://en.wikipedia.org/wiki/E.164 */
export type E164PhoneNumberString = `+${string}`;
```

These types were re-used elsewhere in the codebase, which is causing the current build error:

```
 ./src/app/api/v1/user/welcome-scan/create/route.ts:25:10
Type error: Module '"../../../../../../utils/parse"' has no exported member 'ISO8601DateString'.

  23 | import { setProfileDetails } from "../../../../../../db/tables/onerep_profiles";
  24 | import { StateAbbr } from "../../../../../../utils/states";
> 25 | import { ISO8601DateString } from "../../../../../../utils/parse";
     |          ^
  26 | import { getCountryCode } from "../../../../../functions/server/getCountryCode";
  27 | import { getExperimentationId } from "../../../../../functions/server/getExperimentationId";
  28 | import { getExperiments } from "../../../../../functions/server/getExperiments";
```

So you probably want to re-add those and use those types. (They're not particularly special, but they communicate to the reader that the functions in this file don't just take _any_ string, but strings formatted in a particular way.)